pkg/config/constants.go

```diff
@@ -27,10 +27,12 @@ const (
 
 	NodeAgentRepository = "ibmcom/ibm-node-agent"
 
-	ENVIscsiAgentPort = "ISCSI_AGENT_PORT"
-	ENVEndpoint       = "ENDPOINT"
-	ENVNodeName       = "NODE_NAME"
-	ENVKubeVersion    = "KUBE_VERSION"
+	ENVIscsiAgentPort         = "ISCSI_AGENT_PORT"
+	ENVEndpoint               = "ENDPOINT"
+	ENVNodeName               = "NODE_NAME"
```

Comment:
could you remove these while you're at it?

Reply:
done